src/pages/VenuePage/TemplateWrapper.module.scss

```diff
@@ -4,10 +4,33 @@
 .templateContainer {
   width: 100%;
   height: 100%;
+  background-position: center;
+  background-size: cover;
+  background-repeat: no-repeat;
 }
 
 .shrunk {
   @media (min-width: breakpoint.$md) {
     width: calc(100% - layout.$w-sidebar);
   }
 }
+
+.gradients {
```

Comment:
Not sure if this is a style for `TemplateWrapper` since there is an `<AttendeeLayout />` component, but I guess is OK for the time being if this is simpler until `<TemplateWrapper />` gets fully absorbed by parent